src/screens/route-list/route-list-screen.tsx

```
@@ -361,61 +363,58 @@ export function RouteListScreen() {
+        options,
+        cancelButtonIndex,
+        title: translate("routes.routeActions"),
+        ...getActionSheetStyleOptions(colorScheme),
+      },
+      async (selectedIndex) => {
+        if (selectedIndex === cancelButtonIndex) return
+
+        try {
+          switch (selectedIndex) {
+            case 0: // Add to Calendar
+              const wasAdded = await addRouteToCalendar(routeItem)
```

Biome (2.5.1)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)